### ISSUE TITLE
why aren't  the loss and the accuracy in training stable ? Is it relative with my  disordered data in rgb.txt? 

### ISSUE THREAD

**Issue by MrCuiHao** [USTC-Video-Understanding/I3D_Finetune#27]
**The result in training are as follows:**
step: 1540, loss: 1.3989, accuracy: 0.588 (13.88 sec/batch)
step: 1560, loss: 1.2763, accuracy: 0.625 (12.85 sec/batch)
step: 1580, loss: 2.9938, accuracy: 0.613 (12.66 sec/batch)
step: 1600, loss: 1.3215, accuracy: 0.700 (1.01 sec/batch)
step: 1620, loss: 0.4965, accuracy: 0.713 (17.85 sec/batch)
step: 1640, loss: 1.4940, accuracy: 0.688 (4.32 sec/batch)
step: 1660, loss: 0.4554, accuracy: 0.613 (5.13 sec/batch)
step: 1680, loss: 1.0088, accuracy: 0.662 (6.21 sec/batch)
step: 1700, loss: 0.1014, accuracy: 0.700 (5.01 sec/batch)
step: 1720, loss: 0.2080, accuracy: 0.713 (6.62 sec/batch)
step: 1740, loss: 1.7426, accuracy: 0.725 (3.81 sec/batch)
step: 1760, loss: 1.6926, accuracy: 0.738 (12.97 sec/batch)
step: 1780, loss: 0.3962, accuracy: 0.688 (18.84 sec/batch)
step: 1800, loss: 1.0156, accuracy: 0.738 (3.54 sec/batch)
step: 1820, loss: 0.4570, accuracy: 0.713 (11.37 sec/batch)
step: 1840, loss: 0.1834, accuracy: 0.750 (1.00 sec/batch)
step: 1860, loss: 1.3550, accuracy: 0.600 (5.51 sec/batch)
step: 1880, loss: 0.1034, accuracy: 0.812 (4.54 sec/batch)
step: 1900, loss: 3.2641, accuracy: 0.750 (2.22 sec/batch)
step: 1920, loss: 2.3116, accuracy: 0.637 (13.77 sec/batch)
step: 1940, loss: 0.4947, accuracy: 0.775 (17.61 sec/batch)
step: 1960, loss: 1.2279, accuracy: 0.613 (1.03 sec/batch)
step: 1980, loss: 1.3556, accuracy: 0.650 (1.05 sec/batch)

**the data in rgb.txt，It's disordered ：**

v_HighJump_g25_c04 /media/zoro/ZORO/UCF-101-FRAME/HighJump/v_HighJump_g25_c04 96 39
v_HighJump_g25_c05 /media/zoro/ZORO/UCF-101-FRAME/HighJump/v_HighJump_g25_c05 96 39
v_RopeClimbing_g19_c05 /media/zoro/ZORO/UCF-101-FRAME/RopeClimbing/v_RopeClimbing_g19_c05 164 74
v_YoYo_g12_c04 /media/zoro/ZORO/UCF-101-FRAME/YoYo/v_YoYo_g12_c04 171 100
v_RopeClimbing_g13_c02 /media/zoro/ZORO/UCF-101-FRAME/RopeClimbing/v_RopeClimbing_g13_c02 77 74
v_RopeClimbing_g13_c03 /media/zoro/ZORO/UCF-101-FRAME/RopeClimbing/v_RopeClimbing_g13_c03 77 74
v_RopeClimbing_g13_c01 /media/zoro/ZORO/UCF-101-FRAME/RopeClimbing/v_RopeClimbing_g13_c01 77 74
v_RopeClimbing_g13_c06 /media/zoro/ZORO/UCF-101-FRAME/RopeClimbing/v_RopeClimbing_g13_c06 98 74
v_RopeClimbing_g13_c07 /media/zoro/ZORO/UCF-101-FRAME/RopeClimbing/v_RopeClimbing_g13_c07 84 74
v_RopeClimbing_g13_c04 /media/zoro/ZORO/UCF-101-FRAME/RopeClimbing/v_RopeClimbing_g13_c04 89 74
v_RopeClimbing_g13_c05 /media/zoro/ZORO/UCF-101-FRAME/RopeClimbing/v_RopeClimbing_g13_c05 77 74
v_IceDancing_g04_c02 /media/zoro/ZORO/UCF-101-FRAME/IceDancing/v_IceDancing_g04_c02 249 43
v_IceDancing_g04_c03 /media/zoro/ZORO/UCF-101-FRAME/IceDancing/v_IceDancing_g04_c03 255 43
v_IceDancing_g04_c01 /media/zoro/ZORO/UCF-101-FRAME/IceDancing/v_IceDancing_g04_c01 255 43
v_IceDancing_g04_c06 /media/zoro/ZORO/UCF-101-FRAME/IceDancing/v_IceDancing_g04_c06 261 43
v_HighJump_g25_c02 /media/zoro/ZORO/UCF-101-FRAME/HighJump/v_HighJump_g25_c02 120 39
v_IceDancing_g04_c04 /media/zoro/ZORO/UCF-101-FRAME/IceDancing/v_IceDancing_g04_c04 250 43
v_IceDancing_g04_c05 /media/zoro/ZORO/UCF-101-FRAME/IceDancing/v_IceDancing_g04_c05 249 43
v_Fencing_g25_c01 /media/zoro/ZORO/UCF-101-FRAME/Fencing/v_Fencing_g25_c01 123 27
v_HighJump_g25_c03 /media/zoro/ZORO/UCF-101-FRAME/HighJump/v_HighJump_g25_c03 85 39
v_PlayingTabla_g03_c05 /media/zoro/ZORO/UCF-101-FRAME/PlayingTabla/v_PlayingTabla_g03_c05 254 65
v_CricketShot_g16_c02 /media/zoro/ZORO/UCF-101-FRAME/CricketShot/v_CricketShot_g16_c02 100 23
v_CricketShot_g16_c03 /media/zoro/ZORO/UCF-101-FRAME/CricketShot/v_CricketShot_g16_c03 98 23
v_ThrowDiscus_g19_c01 /media/zoro/ZORO/UCF-101-FRAME/ThrowDiscus/v_ThrowDiscus_g19_c01 68 92
v_CricketShot_g16_c01 /media/zoro/ZORO/UCF-101-FRAME/CricketShot/v_CricketShot_g16_c01 104 23
v_CricketShot_g16_c06 /media/zoro/ZORO/UCF-101-FRAME/CricketShot/v_CricketShot_g16_c06 57 23
v_CricketShot_g16_c07 /media/zoro/ZORO/UCF-101-FRAME/CricketShot/v_CricketShot_g16_c07 50 23
v_CricketShot_g16_c04 /media/zoro/ZORO/UCF-101-FRAME/CricketShot/v_CricketShot_g16_c04 73 23
v_CricketShot_g16_c05 /media/zoro/ZORO/UCF-101-FRAME/CricketShot/v_CricketShot_g16_c05 47 23
v_SoccerPenalty_g04_c02 /media/zoro/ZORO/UCF-101-FRAME/SoccerPenalty/v_SoccerPenalty_g04_c02 72 84
v_CleanAndJerk_g08_c03 /media/zoro/ZORO/UCF-101-FRAME/CleanAndJerk/v_CleanAndJerk_g08_c03 110 20
v_CleanAndJerk_g08_c02 /media/zoro/ZORO/UCF-101-FRAME/CleanAndJerk/v_CleanAndJerk_g08_c02 107 20
v_CleanAndJerk_g08_c01 /media/zoro/ZORO/UCF-101-FRAME/CleanAndJerk/v_CleanAndJerk_g08_c01 236 20
v_CleanAndJerk_g08_c04 /media/zoro/ZORO/UCF-101-FRAME/CleanAndJerk/v_CleanAndJerk_g08_c04 278 20
v_Typing_g19_c01 /media/zoro/ZORO/UCF-101-FRAME/Typing/v_Typing_g19_c01 213 94
v_LongJump_g07_c02 /media/zoro/ZORO/UCF-101-FRAME/LongJump/v_LongJump_g07_c02 171 50
v_LongJump_g07_c03 /media/zoro/ZORO/UCF-101-FRAME/LongJump/v_LongJump_g07_c03 102 50
v_LongJump_g07_c01 /media/zoro/ZORO/UCF-101-FRAME/LongJump/v_LongJump_g07_c01 123 50
v_LongJump_g07_c04 /media/zoro/ZORO/UCF-101-FRAME/LongJump/v_LongJump_g07_c04 82 50
v_LongJump_g07_c05 /media/zoro/ZORO/UCF-101-FRAME/LongJump/v_LongJump_g07_c05 118 50
v_BodyWeightSquats_g18_c03 /media/zoro/ZORO/UCF-101-FRAME/BodyWeightSquats/v_BodyWeightSquats_g18_c03 211 14
v_Surfing_g24_c01 /media/zoro/ZORO/UCF-101-FRAME/Surfing/v_Surfing_g24_c01 249 87
v_HorseRace_g20_c02 /media/zoro/ZORO/UCF-101-FRAME/HorseRace/v_HorseRace_g20_c02 327 40

**Comment by AlexHu123** [comment]
can you inform me of your learning rate? and at first, the accuracy may be not stable, but when the training continues, it may relieve.
